csFastFloat/FastDoubleParser.cs

```diff
@@ -445,13 +445,10 @@ public static unsafe double ParseDouble(byte* first, byte* last, out int charact
     /// <param name="characters_consumed">number of characters consumed while parsing</param>
     /// <param name="expectedFormat">allowed styles for the input string</param>
     /// <param name="decimal_separator">decimal separator to be used</param>
+    /// <param name="value">out : reference to double variable to hold parsed value</param>
```

Comment:
could be spelled out:

"to hold parsed value" -> "to hold the parsed value"

Reply:
done